documentation/modules/migration-plan-options-ui.adoc

```diff
@@ -8,6 +8,7 @@
 
 On the *Plans for virtualization* page of the {ocp} web console, you can click the {kebab} beside a migration plan to access the following options:
 
+* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, you are prompted to download a `.tar` archive file.
```

Comment:
This should be similar to the `Get logs` part in https://github.com/kubev2v/forklift-documentation/pull/384

```suggestion
* *Get logs*: Retrieves the logs of a migration. When you click *Get logs*, a confirmation window opens. After you click *Get logs* in the window, wait until *Get logs*  changes to *Download logs* and then click the button to download the logs.
```